packages/manager/apps/zimbra/public/translations/domains/edit/Messages_fr_FR.json

```diff
@@ -0,0 +1,11 @@
+{
+  "zimbra_domain_edit_modal_title": "Configuration du domaine",
+  "zimbra_domain_edit_modal_content": "Êtes-vous sûr(e) de vouloir supprimer le domaine du serveur Zimbra ?",
+  "zimbra_domain_edit_modal_message_disabled": "Votre domaine est en cours d'utilisation. Vous ne pouvez pas supprimer ce domaine tant qu'il est dans ce cas.",
+  "zimbra_domain_edit_success_message": "La configuration de votre domaine a bien été prise en compte. Elle sera active d'ici quelques instants.",
+  "zimbra_domain_edit_error_message": "La configuration de votre domaine n'a pas pu aboutir. {{ error }}",
```

Comment:
"Merci de réessayer dans quelques instants." is missing at the end of the translation according to the spec

Reply:
No need. We shouldn't have the sentence. Because for example if api return "no granted user", no need retry many time.